src/imgui/ImGuiUtils.cc

```
@@ -283,6 +284,33 @@ std::string getShortCutForCommand(const HotKey& hotkey, std::string_view command
 	return "";
 }
 
+std::string getKeyChordName(ImGuiKeyChord keyChord)
+{
+	int keyCode = ImGuiKey2SDL(ImGuiKey(keyChord & ~ImGuiMod_Mask_));
+	auto scanCode = SDL_GetScancodeFromKey(keyCode);
+	auto name = SDL_GetScancodeName(scanCode);
+
+	return strCat((keyChord & ImGuiMod_Ctrl ? "Ctrl+" : ""),
+		(keyChord & ImGuiMod_Shift ? "Shift+" : ""),
+		(keyChord & ImGuiMod_Alt ? "Alt+" : ""),
+		(keyChord & ImGuiMod_Super ? (ImGui::GetIO().ConfigMacOSXBehaviors ? "Cmd+" : "Super+") : ""),
+		name && std::strlen(name) ? name : "None");
+}
+
+ImGuiKeyChord getKeyChordValue(std::string_view name)
+{
+	ImGuiKeyChord keyMods = {};
+	keyMods |= name.find("Ctrl+")  != std::string_view::npos ? ImGuiMod_Ctrl  : 0;
+	keyMods |= name.find("Shift+") != std::string_view::npos ? ImGuiMod_Shift : 0;
+	keyMods |= name.find("Alt+")   != std::string_view::npos ? ImGuiMod_Alt   : 0;
+	keyMods |= name.find(ImGui::GetIO().ConfigMacOSXBehaviors ? "Cmd+" : "Super+") != std::string::npos ? ImGuiMod_Super : 0;
+	// get the substring after the '+' separator in the expected (mod '+')*key format
+	auto pos = name.find_last_of('+');
+	std::string suffix = std::string(pos == std::string_view::npos ? name : name.substr(pos + 1));
+	SDL_Keycode keyCode = SDL_GetKeyFromName(suffix.c_str());
+	return SDLKey2ImGui(keyCode) | (keyCode != SDLK_UNKNOWN ? keyMods : 0);
+}
```

Details:

I think this happens to work fine for `None`, but it might be useful to more explicitly handle this, or else add a comment explaining how `None` is handled now.  (Because I first thought we didn't handle it correctly).

Do we want to add more robust error handling here? For example:
* Typos in the modifiers are now silently ignored.
* Typos in the key name itself are now interpreted as `None`.